.github/workflows/go.yml

```diff
@@ -18,8 +18,5 @@ jobs:
       with:
         go-version: 1.17
 
-    - name: Build
-      run: go build -v ./...
-
     - name: Test
       run: go test -v ./...
```

Comment:
this is great - ultimately, since some tests ping NSI api and NASS CDL api's we need to skip those tests (since they will not be credentialed on the build server)